src/routes/profile-route/profile-recommended-route.jsx

```diff
@@ -0,0 +1,237 @@
+import { useQuery } from "@apollo/client";
+import { useMemo, useState } from "react";
+import { useMediaQuery } from "react-responsive";
+import { Link } from "react-router-dom";
+import { Footer, Header, MovieCardList } from "../../components";
+import { GET_MOVIES, GET_USERS_WATCHLIST } from "../../graphql";
+import arrowdown from "../../images/arrowdown.svg";
+import arrowup from "../../images/arrowup.svg";
+import sloppreferances from "../../images/sloppreferances.svg";
+import { ProfileHorizontalMenu } from "./profile-horizontal-menu";
+import { ProfileSidebar } from "./profile-sidebar";
+
+export const ProfileRecommendedRoute = () => {
```

Comment:
Everything looks good, but I believe that something was misinterpreted or miscommunicated. This logic seems to be more for the me goblin page rather than the recommended page. This code might only have to be slightly refactored for recommending slops. It might be blocked until the user preferences have been created.